src/pages/invoice.js

```diff
@@ -383,6 +384,9 @@ export class InvoicePage extends React.Component {
                   </Flex>
                 )}
               </Box>
+              <P mt={2} textAlign="left" color="black" padding="4rem 0">
+                {invoice.extraInfo}
+              </P>
```

Comment:
Extra info may not be there. Let's be safe:

```suggestion
              {invoice.extraInfo && (
                <P mt={2} textAlign="left" color="black" padding="4rem 0">
                  {invoice.extraInfo}
                </P>
              )}
```